state/spacesdiscovery.go

```diff
@@ -29,35 +26,6 @@ func (st *State) getModelSubnets() (set.Strings, error) {
 	return modelSubnetIds, nil
 }
 
-// ReloadSpaces loads spaces and subnets from provider specified by environ into state.
-// Currently it's an append-only operation, no spaces/subnets are deleted.
-func (st *State) ReloadSpaces(environ environs.BootstrapEnviron) error {
```

Comment:
This separate file is probably not required now. Just poke the methods into the subnets/spaces files.

`SaveProviderSpaces` and `SaveProviderSubnets` are better canonical names too, but feel free to ignore my pedantry there.